builtins/en/steps/implementation-high-write-tests-to-implement.yaml

```
@@ -0,0 +1,12 @@
+name: write_tests
+tags: [ coding, test-planning ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)